### DISCUSSION TITLE
Is it possible to add a captcha to login pages?

### DISCUSSION THREAD

**Discussion by Chaosxmk** [joomla/joomla-cms#40358] [Q&A]
I noticed that even after integrating a captcha into my website, the login pages don't have one. Typically, this wouldn't be an issue but a client has asked me to add one to login pages.

**Comment by webgras** [comment]
Just found your question, if it is still open...
Right now you can add a captcha only to the registration page, but not the login page.

**Reply by Chaosxmk** [reply]
I opened an issue about this a bit after asking and was assured that the login page not having one was standard procedure and none would be added.

**Reply by babakhakimi** [reply]
you can do this with a plugin
It is not a complicated task
